BaboViolent2/Code/CMaster.cpp

```diff
@@ -282,7 +285,7 @@ void CMaster::update(float in_delay)
 void CMaster::ReceivePeersPacket()
 {
 	char *buf=0;
-	long peerID;
+	INT4 peerID;
```

Comment:
INT4 comes from platform_types.h and originally it used in baboNet code so I have to update all places to match types.